client/my-sites/sidebar/sidebar.jsx

```diff
@@ -717,7 +717,18 @@ export class MySitesSidebar extends Component {
 
 	renderSidebarMenus() {
 		if ( this.props.isDomainOnly ) {
-			return null;
+			return (
```

Comment:
I think this is fine for now but we'll probably want a `renderDomainOnlySidebar` function we call here if the domain-only sidebar gets any bigger.